src/component/mxgraph/StyleConfigurator.ts

```diff
@@ -125,8 +127,9 @@ export default class StyleConfigurator {
   }
 
   private configureTaskStyle(): void {
-    const style = this.mxUtils.clone(this.getStylesheet().getCellStyle(ShapeBpmnElementKind.TASK_USER), this.getDefaultVertexStyle());
-    style[this.mxConstants.STYLE_STROKECOLOR] = '#663399';
+    const style = this.cloneDefaultVertexStyle();
```

Comment:
:question: I guess we will generalize styling for other tasks when implementing rendering for user or service task

Reply:
Yes, I started the refactoring in the PR of the Service Task #208 :wink: